app/views/membership_applications/show.html.haml

```diff
@@ -8,3 +8,21 @@
   - if @membership_application.phone_number
     %p= @membership_application.phone_number
   %p= mail_to @membership_application.company_email, @membership_application.company_email
+  .wpcf7
+    = form_for @membership_application, url: membership_application_path do |f|
+      - if @membership_application.status.to_i == 2
+        - chosen = 2
+      - elsif @membership_application.status.to_i == 3
+        - chosen = 3
+      - else
+        - chosen = 1
+      = f.select(:status, options_for_select([['Pending', 1],
+                                              ['Accepted', 2],
+                                              ['Rejected', 3]],
+                                              chosen))
```

Comment:
options_for_select will set the currently-selected options based on the current value of select field (:status).  So, lines 19-21 could be:
```
== f.select(:status, [['Pending', 'Pending'],
                      ['Accepted', 'Accepted'],
                      ['Rejected', 'Rejected']])
```
And lines 13-18 can be removed.